Description: Find rubiks binaries
 They have different names in Debian.
Author: Tobias Hansen <thansen@debian.org>

--- a/sage/src/sage/interfaces/rubik.py
+++ b/sage/src/sage/interfaces/rubik.py
@@ -90,7 +90,7 @@
     """
     Interface to Michael Reid's optimal Rubik's Cube solver.
     """
-    __cmd = "optimal"
+    __cmd = "rubiks_optimal"
 
     def __init__(self, verbose=False, wait=True):
         self.verbose = verbose
@@ -178,7 +178,7 @@
 
 class CubexSolver:
 
-    __cmd = "cubex"
+    __cmd = "rubiks_cubex"
 
     def __call__(self, facets):
         return self.solve(facets)
@@ -232,7 +232,7 @@
 
 class DikSolver:
 
-    __cmd = "dikcube"
+    __cmd = "rubiks_dikcube"
 
     def __call__(self, facets):
         return self.solve(facets)
